server.go

```diff
@@ -311,3 +312,71 @@ func (s *server) ListLimits(ctx context.Context,
 		Limits:       rpcLimits,
 	}, nil
 }
+
+func (s *server) ListForwardingHistory(ctx context.Context,
+	req *circuitbreakerrpc.ListForwardingHistoryRequest) (
+	*circuitbreakerrpc.ListForwardingHistoryResponse, error) {
+
+	var (
+		// By default query from the epoch until now.
+		startTime = time.Time{}
+		endTime   = time.Now()
```

Comment:
I wonder if it is cleaner to have open-ended queries using nil. This works too though.

Reply:
I think it makes the db logic _a little_ messier needing to check nil values so would rather leave as-is and update if we want different types of queries.